### ISSUE TITLE
Remediated Document 

### ISSUE THREAD

**Issue by cmathers1** [ONRR/onrr.gov-site#936]
Good morning, 

Can you please replace the existing document on onrr.gov with the attached remediated document that is Section 508 Compliant. Thank you!
[911220.pdf](https://github.com/ONRR/onrr.gov-site/files/7001197/911220.pdf)

**Comment by lpgoldstein** [comment]
@cmathers1 I've replaced the pdf in the dev site, please check: http://ONRR2017/reportpay/PDFDocs/911220.pdf
http://2017onrr/DearRep.htm

**Comment by cmathers1** [comment]
Good morning,

I am having trouble accessing the link. Thank you.

Sincerely,

Christina

Christina Mathers
Staff Assistant
Director/Deputy Director's Office
Office of Natural Resources Revenue
6th Avenue and Kipling Street Bldg. 85
Denver, Colorado 80225
Phone:  303-231-3429
Cell: 303-503-1315
________________________________
From: lpgoldstein ***@***.***>
Sent: Wednesday, August 18, 2021 4:14 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Mathers, Christina ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Remediated Document (#936)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@cmathers1<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcmathers1&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C6f3108bfb9a043c78c2f08d962310af2%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648785044467648%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=N6Yl6KiYLC6crTHxV1QksU6vhy0awWLlfb10KTsLvUM%3D&reserved=0> I've replaced the pdf in the dev site, please check: http://ONRR2017/reportpay/PDFDocs/911220.pdf
http://2017onrr/DearRep.htm

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F936%23issuecomment-900993836&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C6f3108bfb9a043c78c2f08d962310af2%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648785044467648%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=E%2F6ByYOWoKCay6KTrN45hykHsIRKdnGTFRtD1Z1nBG8%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAU5GF7Q5UC4HUGCF65LVMBDT5OBZVANCNFSM5CKDNWNA&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C6f3108bfb9a043c78c2f08d962310af2%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648785044477607%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=S8mv2PvyHqNdZHD5QneGTH4QN6OsAVjZpxHidueEk58%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C6f3108bfb9a043c78c2f08d962310af2%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648785044487568%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qV5ZqSciWcUy5vHjYBT09u88UtNOHBrhif6PYzsN6kQ%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C6f3108bfb9a043c78c2f08d962310af2%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648785044487568%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=x3JOVwo7HvgSsY1K7MNu2LW6p7c5qUu8Csw2O%2FMXA5Y%3D&reserved=0>.

**Comment by lpgoldstein** [comment]
@cmathers1 Please check again now - it's working for me http://2017onrr/reportpay/PDFDocs/911220.pdf

**Comment by cmathers1** [comment]
Thank you! Yes, I can see it now. We are good.

Sincerely,

Christina

Christina Mathers
Staff Assistant
Director/Deputy Director's Office
Office of Natural Resources Revenue
6th Avenue and Kipling Street Bldg. 85
Denver, Colorado 80225
Phone:  303-231-3429
Cell: 303-503-1315
________________________________
From: lpgoldstein ***@***.***>
Sent: Wednesday, August 18, 2021 10:36 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Mathers, Christina ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Remediated Document (#936)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@cmathers1<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcmathers1&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ce9191b23db314a9c895508d962666a3d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649014280228005%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=097bJ44xvsuqXNE5PtfGvjetWyxe0pqqx7M15bs01ys%3D&reserved=0> Please check again now - it's working for me http://2017onrr/reportpay/PDFDocs/911220.pdf

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F936%23issuecomment-901261016&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ce9191b23db314a9c895508d962666a3d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649014280228005%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=SOtXdXNdprV0OLsz9OKvROFh5Wbz2IlgFquzZirlw8g%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAU5GF7VXDWBSGCD6M2CGT2LT5PORXANCNFSM5CKDNWNA&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ce9191b23db314a9c895508d962666a3d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649014280237962%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3CojDTGXnTnuqOuSyhec9RHetsZUnuPRbqLnfNsBKvM%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ce9191b23db314a9c895508d962666a3d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649014280237962%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=S%2FbvAmsbvQOp29IgMeJiQwRQ%2BI82D8f2sSIG%2BbdFW3Q%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ce9191b23db314a9c895508d962666a3d%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649014280247918%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=YODAX7vcTxJEiJpXchA3wauOkQccUEwlS8Eg5J4mcbc%3D&reserved=0>.

**Comment by lpgoldstein** [comment]
@cmathers1 This should be live soon
